Source/ASNetworkImageNode.mm

```diff
@@ -353,8 +355,18 @@ - (void)displayWillStartAsynchronously:(BOOL)asynchronously
         if (result) {
           [self _locked_setCurrentImageQuality:1.0];
           [self _locked__setImage:result];
-          
           _imageLoaded = YES;
+
+          // Call out to the delegate.
+          if (_delegateFlags.delegateDidLoadImageWithInfo) {
+            ASDN::MutexUnlocker l(__instanceLock__);
+            ASNetworkImageNodeDidLoadInfo info = {};
+            info.imageSource = ASNetworkImageSourceSynchronousCache;
+            [_delegate imageNode:self didLoadImage:result info:info];
+          } else if (_delegateFlags.delegateDidLoadImage) {
+            ASDN::MutexUnlocker l(__instanceLock__);
+            [_delegate imageNode:self didLoadImage:result];
+          }
```

Comment:
We previously weren't calling out to the delegate at all in this synchronous fetch scenario. I think we should.

Reply:
Good catch!